MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponent.java

```diff
@@ -1704,7 +1723,18 @@ private void updateAccuracyRadius(Location location, boolean noAnimation) {
     } else {
       radius = Utils.calculateZoomLevelRadius(mapboxMap, location);
     }
-    locationAnimatorCoordinator.feedNewAccuracyRadius(radius, noAnimation);
+    if(forceUpdate){
+      previousRadius = radius;
+      locationAnimatorCoordinator.feedNewAccuracyRadius(radius, noAnimation);
+    }else {
+
+      //Draw only if new radius size changes when taking pixel tolerance into account
+      float approximatedRadius = radius - (radius % options.locationCircleRadiusTolerance());
```

Comment:
Can you elaborate on the logic here? If  the value of `locationCircleRadiusTolerance` is default, which is 1, then `approximatedRadius` will be 0, right?

Reply:
It can be changed to something else, but after playing a bit with it i found this was acceptable.
(Any suggestion welcome)
The idea was: if tolerance is for instance 5 pixels
A radius of 3.45 would be approximated to 3.45 - (3.45 % 5) = 0
A radius of 8.17 would be approximated to 8.17 - (8.17 % 5) = 5
And so on with approximatedRadius that can only be = 0 / 5 / 10 / 15 / 20 ....

The idea is to have a reduced number of possible radius, that are multiple of the tolerance.
This way, the approximation is not so far from the real radius, and the reduced number of approximatedRadius would lead to less redraw.